frontend/src/utils/date.ts

```diff
@@ -160,3 +160,39 @@ export const transformCheckedBoxesValueToInvalidDays = (
   const validDaysSet = new Set(validDays)
   return ALL_INVALID_DAYS_ARR.filter((day) => !validDaysSet.has(day))
 }
+
+const DAY_TO_NUMBER_MAP: Record<InvalidDaysOptions, number> = {
```

Comment:
can we not use the constant previously created? this seems like a duplicate constant